release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -499,6 +499,12 @@ For more information, see xref:../networking/metallb/about-metallb.adoc#about-me
 The `route.openshift.io/destination-ca-certificate-secret` annotation can now be used on an Ingress object to define a route with a custom certificate (CA).
 
 See xref:../networking/routes/route-configuration.adoc#nw-ingress-creating-a-route-via-an-ingress_route-configuration[Creating a route using the destination CA certificate in the Ingress annotation] for more information.
+[id="ocp-4-11-support-for-hosted-control-planes"]
+==== Hosted control planes (Technology Preview)
+
+With hosted control planes for {product-title}, you can host clusters at scale to reduce management costs, optimize cluster deployment time, and separate management and workload concerns. You can enable this deployment model as a Technology Preview feature when you install the multicluster engine for Kubernetes operator version 2.0. For more information, see xref:../architecture/control-plane.adoc#hosted-control-planes-overview_control-plane[Overview of hosted control planes (Technology Preview)].
```

Comment:
```suggestion
With hosted control planes for {product-title}, you can host clusters at scale to reduce management costs, optimize cluster deployment time, and separate management and workload concerns. You can enable this deployment model as a Technology Preview feature when you install the multicluster engine for Kubernetes Operator version 2.0. For more information, see xref:../architecture/control-plane.adoc#hosted-control-planes-overview_control-plane[Overview of hosted control planes (Technology Preview)].
```
Operator is always [capitalized](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#operator).